clair/clair.go

```diff
@@ -0,0 +1,209 @@
+package clair
+
+import (
+	"os"
+	"strings"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/containerops/wrench/setting"
+	"github.com/coreos/clair/database"
+	"github.com/coreos/clair/updater"
+	"github.com/coreos/clair/utils"
+	"github.com/coreos/clair/utils/types"
+	"github.com/coreos/clair/worker"
+	"github.com/coreos/pkg/capnslog"
+
+	_ "github.com/coreos/clair/updater/fetchers"
+	_ "github.com/coreos/clair/worker/detectors/os"
+	_ "github.com/coreos/clair/worker/detectors/packages"
+)
+
+type ClairConfig struct {
+	KeepDB       bool
+	DBPath       string
+	LogLevel     string
+	Duration     string
+	VulnPriority string
+}
+
+const (
+	DefaultClairUpdateDuration = "1h0m0s"
+	DefaultClairLogLevel       = "info"
+	DefaultClairDBPath         = "/db"
+	DefaultClairVulnPriority   = "Low"
+)
+
+var (
+	clairConf    ClairConfig
+	clairStopper *utils.Stopper
+)
+
+func init() {
+	var conf ClairConfig
+	// Load database setting
+	if setting.ClairDBPath != "" {
```

Comment:
some problem occured because init function will be called before reading config file,please check it.